static/app/views/eventsV2/table/columnEditCollection.tsx

```diff
@@ -610,4 +706,32 @@ const StyledSectionHeading = styled(SectionHeading)`
   margin: 0;
 `;
 
+const AliasInput = styled(Input)`
+  /* Match the height of the select boxes */
+  height: 40px;
+  min-width: 50px;
+`;
+
+const AliasField = styled('div')<{singleColumn: boolean}>`
+  margin-top: ${space(1)};
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
+    margin-top: 0;
+    margin-left: ${space(1)};
+  }
+
+  @media (max-width: ${p => p.theme.breakpoints[0]}) {
+    grid-row: 2/2;
+    grid-column: ${p => (p.singleColumn ? '1/-1' : '2/2')};
```

Comment:
Unless this is some grid trick, then you should
```suggestion
    grid-column: ${p => (p.singleColumn ? '1/1' : '2/2')};
```

Reply:
I would like it to take the whole space so it has to be `1/-1`